src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,113 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationService registrationService = new RegistrationServiceImpl();
+    private StorageDao storageDao = new StorageDaoImpl();
+    private User jac;
+    private User daniel;
+
+    @BeforeEach
+    void setUp() {
+        jac = new User();
+        jac.setLogin("Jac");
+        jac.setAge(19);
+        jac.setPassword("1245787");
+        daniel = new User();
+        daniel.setLogin("Daniel");
+        daniel.setAge(22);
+        daniel.setPassword("1245787");
+    }
+
+    @Test
+    void register_nullAge_notOk() {
+        jac.setAge(null);
+        RegistrationException thrown = assertThrows(RegistrationException.class, () -> {
+            registrationService.register(jac);
+        });
+        assertEquals("Login can't be null", thrown.getMessage());
+    }
+
+    @Test
+    void register_18Age_itsOk() {

```

Comment:
dont put concrete values into method names - use abstract names
minimal age can be changed potentially